src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,54 @@
+package nextstep.subway.domain;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import nextstep.subway.presentation.LineRequest;
+
+import javax.persistence.*;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@Entity
+public class Line {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(length = 20, nullable = false)
+    private String name;
+
+    @Column(length = 20, nullable = false)
+    private String color;
+
+    @Embedded
+    private Sections sections = new Sections();
+
+    public Line(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
+
+    public static Line createLine(Station upStation, Station downStation, LineRequest lineRequest) {
+        Line createdLine = new Line(lineRequest.getName(), lineRequest.getColor());
+
+        Sections sections = createdLine.getSections();
+        Section requestSection = Section.createSection(
+                createdLine,
+                upStation,
+                downStation,
+                lineRequest.getDistance()
+        );
+        sections.addSections(requestSection);
```

Comment:
> 이미 앞선 미션에서 Sections 클래스를 생성하고 여기서 구간의 생성/삭제의 책임을 넘겼는데, 이번 미션에서 다시 Line이 그 책임을 가져가야 하는 것처럼 보여 좀 혼란 스러웠습니다.

`Line`과 `Section`의 연관관계를 고려해서 아래와 같이 작성해보면 어떨까요?
```suggestion
        createdLine.addSection(Section.createSection(
            createdLine, upStation, downStation, lineRequest.getDistance()
        ));
```

Reply:
구간을 추가하는 것은 노선이 처리하도록 하고 구간 추가의 검증이나 실제 추가 행위는 구간 리스트의 래퍼 클래스가 담당하도록 변경해보았습니다.